hummingbot/connector/exchange/kraken/kraken_in_flight_order.pyx

```diff
@@ -86,6 +94,11 @@ cdef class KrakenInFlightOrder(InFlightOrderBase):
             "userref": self.userref
         }
 
+    def update_exchange_order_id(self, exchange_id: str):
+        super().update_exchange_order_id(exchange_id)
+        if self.is_local:
+            self.last_state = "pending"
```

Comment:
Before the PR the initial status was "NEW". Now, the status for the confirmed order in the exchange is "pending". Is that correct?

Reply:
Ah yes, I see your point. Leaving it as "local" and letting the ws order status update do its work

Reply:
What I expected was to see we were configuring the status as NEW once we receive the exchange_order_id, instead of "pending", since before this PR the initial status was NEW and not "pending"

Reply:
I looked around but there doesn't seem to be any use of "NEW" other than being the previous default value. Now "NEW" is essentially replaced with "local", which is more in line with what we have in the previous integrations